### DISCUSSION TITLE
Documentation Plan

### DISCUSSION THREAD

**Discussion by ralfHielscher** [mtex-toolbox/mtex#582] [Feature Requests]
We are very proud of our large and extensive [documentation](https://mtex-toolbox.github.io/Documentation.html). However, documenting a project like MTEX is a challenging and ongoing task. Therefore we are extremely happy about any help, e.g., spelling fixes, examples, theoretical explanations, special use cases, etc. [Contributing to the documentation is extremely simple](https://mtex-toolbox.github.io/Documentation.html#1) as all the documentation pages are essentially Matlab scripts that are included within MTEX. They can be accessed by
```matlab
edit OrientationDefinition.m
```
where `OrientationDefinition` is just a placeholder for the specific site you are interested in. In this post I keep a list of topics that require more documentation and a list of recently updated documentation pages. Everybody is invited to shorten this list by adding to the documentation or to extending this list if you think a topic requires more documentation. Please comment below if you think a topic is not yet sufficiently covered in the documentation.

- [ ] [Grain Tutorial](https://mtex-toolbox.github.io/GrainTutorial.html)
- [ ] plot orientation / rotation colored by property
- [ ] [Misorientation Theory](https://mtex-toolbox.github.io/MisorientationTheory.html)  
- [ ] [Misorientation Twinning](https://mtex-toolbox.github.io/Twinning.html)
- [ ] [Misorientation Distribution Function](https://mtex-toolbox.github.io/MisorientationDistributionFunction.html)
- [ ] [Axis Distribution Function](https://mtex-toolbox.github.io/AxisDistributionFunction.html)
- [ ] [Angle Distribution Function](https://mtex-toolbox.github.io/AngleDistributionFunction.html)
- [ ] [Orientation Export](https://mtex-toolbox.github.io/OrientationExport.html)
- [ ] EBSD Export - conventions

**Comment by nyyssont** [comment]
The documentation on exporting CTF files could be expanded.

https://mtex-toolbox.github.io/EBSDExport.html

I, for one, can't seem to get the orientations sorted out correctly when importing a CTF to MTEX and exporting back to CTF. A simple example would help a lot.

If / when I manage to get the export working the way I want I can try to write something. This might require adding a few images separately to a server just for illustration purposes, is that possible to do?

-Tuomo

**Reply by ralfHielscher** [reply]
Hi Tuomo,

additional images are no problem. Just put them into the directory of the script file and include them as
```matlab
% 
% <<image.png>>
% 
```
Ralf.
